DiamondLightSource/python-artemis
# 355 move zebra logic to plan

Fixes #355

Allows us to set up the zebra in the experimental plan in order to customise it for different experiments

### To test:
1. Verify this makes sense
2. Check tests pass (ideally with S03 and script for slit gaps)
